defaults/main.yml

```diff
@@ -172,7 +172,8 @@ sonar_ce_worker_count: 1
 #    is not enabled by default on your environment:
 #    http://docs.oracle.com/javase/8/docs/technotes/guides/vm/server-class.html
 #
-sonar_search_java_opts: "-Xmx1G -Xms256m -Djava.net.preferIPv4Stack=true -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=75 -XX:+UseCMSInitiatingOccupancyOnly -XX:+HeapDumpOnOutOfMemoryError"
+# Defaults:
+# sonar_search_java_opts: "-Xmx1G -Xms1G -Xss256k -Djava.net.preferIPv4Stack=true -XX:CMSInitiatingOccupancyFraction=75 -XX:+UseCMSInitiatingOccupancyOnly -XX:+HeapDumpOnOutOfMemoryError"
```

Comment:
I think you could add True as second parameter to the matching default(...) in the template and set that variable to nothing instead of commenting it.
I'm not fan of commented variables.

Reply:
Done as requested